app/views/account/check-your-emails.html

```diff
@@ -0,0 +1,56 @@
+
+{% extends "_templates/page-template.html" %}
+
+{% set pageHeading = "Check your email" %}
+{% set formAction = "./check-email" %}
+{% set signedOut = "true" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds-from-desktop">
+
+      <h1 class="govuk-heading-l">Check your email</h1>
+      {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+      {% set insetTextHtml %}
+        We have sent an email to: <br>
+        <span class="govuk-body-l govuk-!-font-weight-bold">{{ data.checkEmail }}</span>
+      {% endset %}
+
+      {{ govukInsetText({
+        html: insetTextHtml
+      }) }}
+
+      <p class="govuk-body">This email contains a 6-digit code.</p>
+      <p class="govuk-body">The email might take a few minutes to arrive. If you do not get an email, check your spam folder.</p>
+      <p class="govuk-body">Your code will expire after 15 minutes.</p>
+
+      {{ govukInput({
+        label: {
+        text: "Enter the 6 digit security code"
+        },
+        classes: "govuk-input--width-5",
+        id: "width-6",
+        name: "width-6"
+      }) }}
+
+      <p class="govuk-body"> <a href="/application/sorry" class="govuk-link">Request a new code</a> if:</p>
```

Comment:
This link should go to this file but lets also rename that file to something more logical like `request-code` 

```suggestion
      <p class="govuk-body"> <a href="/account/create-account" class="govuk-link">Request a new code</a> if:</p>
```